core-processor/src/common.rs

```diff
@@ -52,11 +52,14 @@ pub struct DispatchResult {
     /// Original dispatch.
     pub dispatch: Dispatch,
 
-    /// List of generated outgoing messages.
-    pub outgoing: Vec<Dispatch>,
+    /// List of generated from program messages.
```

Comment:
May be there messages not from program? A bit ambiguous docs here. What were the main reason to rename?

Reply:
Can we have any messages created after executions which are not from program?
I renamed it, because of possible misconception with `core::OutgoingMessage`, which could be considered by reader as the only content of  `DispatchResult.outgoing`. So by that I declare that there could be generated any kind of dispatches after execution.

Reply:
Got it